src/predict/tests/test_datamanagement.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import sys
+sys.path.append('./')
+from models.datamanagement import get_channel_data_raw, save_predictions, get_channel_best_configurations
+from models.datamanagement import calculate_hourly_averages, get_channel_id, get_all_coordinates
```

Comment:
For unit tests, could we mock the data and not make a direct call to the actual database? More notes here:[ 1](https://stackoverflow.com/questions/51988331/using-a-fake-mongodb-for-pytest-testing) and [2](https://pypi.org/project/pytest-mongodb/)